### ISSUE TITLE
Error while updating property 'selectionColor' of a view managed by: AndroidTextInput

### ISSUE THREAD

**Issue by eliasturbay** [react-native-training/react-native-elements#331]
When using this component (only in Android)
```
<SearchBar
          lightTheme
          placeholder='Search...' 
          containerStyle={{backgroundColor: Colors.tintColor, borderTopWidth: 0, borderBottomWidth: 1}}
          inputStyle={{backgroundColor: 'white'}}
        />
```

Version: "react-native-elements": "^0.11.1"

Complete dependecies list:
```
  "dependencies": {
    "@expo/react-native-navigator": "^0.4.1",
    "@expo/react-native-read-more-text": "^1.1.0",
    "@expo/vector-icons": "~4.0.0",
    "expo": "^16.0.0",
    "hoist-non-react-statics": "^1.2.0",
    "moment": "^2.18.1",
    "react": "16.0.0-alpha.6",
    "react-addons-shallow-compare": "^15.5.2",
    "react-native": "https://github.com/exponent/react-native/archive/sdk-16.0.0.tar.gz",
    "react-native-app-intro": "~1.1.5",
    "react-native-button": "~1.8.2",
    "react-native-communications": "^2.2.1",
    "react-native-dropdownalert": "^2.7.1",
    "react-native-elements": "^0.11.1",
    "react-native-grid-component": "^1.0.6",
    "react-native-i18n": "^1.0.0",
    "react-native-textinput-effects": "^0.3.0",
    "react-navigation": "^1.0.0-beta.7",
    "react-navigation-addons": "git://github.com/satya164/react-navigation-addons.git#master"
  }
```

This issue happened when upgrading to expo 16

![img_3527](https://cloud.githubusercontent.com/assets/314122/25315755/c1fbc41e-2830-11e7-816e-208ac1a590d8.JPG)

**Comment by tarang9211** [comment]
Hey @eliasturbay thanks for providing detailed information regarding the issue! When you get a chance, have a look at https://github.com/react-native-training/react-native-elements/issues/319 and let me know if that helps.

**Comment by eliasturbay** [comment]
Hey @tarang9211 thanks for your quick response. I checked that issue and the others related. Correct me if I'm wrong... I'll have to wait for an update in React Native code base to have a fix for this issue?

**Comment by tarang9211** [comment]
@eliasturbay Yup, I think that's what will have to happen. @bozzmob Does that sound about right?

**Comment by martinezguillaume** [comment]
Hey ! Any workaround here ? I'm using expo ! Thanks !

**Comment by RyszardRzepa** [comment]
I updated to react-native -v 0.44 , but still having this Error "Error while updating property 'selectionColor' of a view managed by: AndroidTextInput"

**Comment by RyszardRzepa** [comment]
I found this error related to { searchBar }. After removing this component, error disappear.

**Comment by naveenkonduru** [comment]
it didn't work even in Exp 17

**Comment by itsashis4u** [comment]
Same as @naveenkonduru

**Comment by martinezguillaume** [comment]
Same here... With RN 0.44

**Comment by Monte9** [comment]
We are looking into this. Sorry about that. 

We will fix this asap and push a fix for it soon.
/cc @binoy14

**Comment by vthu** [comment]
@Monte9 
Thanks for taking a look at this issue...
Hoping to get a solution soon :)

**Comment by vthu** [comment]
@binoy14 @Monte9 Any update on this bug?

**Comment by vthu** [comment]
Thanks @binoy14 :)

**Comment by hskb10** [comment]
Hi @binoy14!, @Monte9, @vthu update on the bug? Thanks!

**Comment by binoy14** [comment]
I can confirm this only happens if you are using Expo in any way, either the expo client or create-react-native-app. To solve this issue immediately, I am removing the default selectionColor. It can always be added by passing it as a prop to FormInput and SearchBar. Can someone confirm this works? Change the react-native-elements in package.json to following

```
"react-native-elements": "git://github.com/react-native-training/react-native-elements.git#next"
```

**Comment by rafaelmhp** [comment]
@binoy14 I was having the same problem but I can confirm the fix worked after the update. Thanks!

**Comment by xavier-villelegier** [comment]
@binoy14 Same here, fixed ! Thanks

**Comment by Monte9** [comment]
Thanks for the fix @binoy14!

Closing this issue as this fix will be released in `v0.12.3`

**Comment by NunoCardoso** [comment]
I am having this issue, and I am with the 0.17 version.

**Comment by lejoss** [comment]
hi, i'm having the same issue, i'm using expo too. I detached my app, opened it on Android Studio and got the error.

`"expo": "^21.0.0"`
`"react-native-elements": "^0.17.0"`
`"react": "16.0.0-alpha.12"`

```
{
  "expo": {
    "name": "mobile",
    "description": "No description",
    "slug": "mobile",
    "privacy": "public",
    "sdkVersion": "21.0.0",
    "androidStatusBar": {
      "barStyle": "dark-content",
      "backgroundColor": "#0D4969"
    },
    "loading": {
      "backgroundImage": "./assets/images/marmon.png",
      "hideExponentText": true
    },
    "version": "1.0.0",
    "orientation": "portrait",
    "primaryColor": "#cccccc",
    "icon": "./assets/icons/app-icon.png",
    "notification": {
      "icon": "./assets/icons/notification-icon.png",
      "color": "#000000"
    },
    "packagerOpts": {
      "assetExts": ["ttf"]
    },
    "ios": {
      "bundleIdentifier": "com.sk3vy.new-project-template",
      "supportsTablet": true
    }
  }
}
```

how can i debug the source of the issue?

**Comment by iRoachie** [comment]
Hey all, a fix was made in https://github.com/react-native-training/react-native-elements/pull/666 to fix this problem. However the version hasn't been released as yet. You can use the `next` branch of react-native-elements in the mean time until 0.18 is released.

**Comment by lejoss** [comment]
i was trying the next version on RNE, to avoid the `selectionColor` issue, but i got this instead

![error](https://user-images.githubusercontent.com/11046621/32191663-e0bca276-bd7f-11e7-9245-06aee9a798c8.jpg)

these are my dependencies

`git+https://github.com/react-native-training/react-native-elements.git#next`
```
"expo": "^22.0.0",
    "react": "16.0.0-beta.5",
    "react-native": "https://github.com/expo/react-native/archive/sdk-22.0.1.tar.gz",
    "react-native-elements": "git+https://github.com/react-native-training/react-native-elements.git#next"
```

**Comment by xavier-villelegier** [comment]
@lejoss What if you run `npm install @expo/vector-icons --save-dev` ?

**Comment by lejoss** [comment]
@xavier-villelegier yes you were right, i missed the module, thanks. FormInput is working now.

<img width="385" alt="screen shot 2017-10-30 at 3 21 08 pm" src="https://user-images.githubusercontent.com/11046621/32193687-10ed1740-bd86-11e7-98f0-0707e7869c69.png">
```
"dependencies": {
    "expo": "^22.0.0",
    "react": "16.0.0-beta.5",
    "react-native": "https://github.com/expo/react-native/archive/sdk-22.0.1.tar.gz",
    "react-native-elements": "git+https://github.com/react-native-training/react-native-elements.git#next"
  },
  "devDependencies": {
    "@expo/vector-icons": "^6.1.0"
  }
```

**Comment by titkov5** [comment]
Got the same issue with TextField, but the problem was inside tintColor TextInput of RN (0.49.3) ^ i found my problem in AndroidManifest.xml this is my lines http://prntscr.com/h48lwb

**Comment by iRoachie** [comment]
Fixed in v18.0.0

**Comment by mikeumus** [comment]
@lilymGoh's [solution](https://github.com/n4kz/react-native-material-textfield/issues/54#issuecomment-340553507) worked for me here:

> Remove `selectionColor={...}` on line 484 in `src/components/field/index.js` should fix it.
